tools/update-client

```diff
@@ -1,38 +0,0 @@
-#!/bin/sh
```

Comment:
This script was removed because updating the client is now done via a Jenkins build with the `BUILD_TYPE` parameter set to `update-hypothesis-client`.